spec/engine/miq_ae_method_service/miq_ae_service_model_base_spec.rb

```diff
@@ -94,6 +94,21 @@
     end
   end
 
+  describe 'YAML#load' do
+    let(:service)     { FactoryBot.create(:service, :name => 'test_service') }
+    let(:svc_service) { MiqAeMethodService::MiqAeServiceService.find(service.id) }
+
+    it 'loads invalid svc_model for objects without related ar_model' do
+      yaml = svc_service.to_yaml
+      service.delete
+      model_from_yaml = YAML.safe_load(yaml)
+      expect { model_from_yaml.reload }.to raise_error(
+        NoMethodError,
+        "undefined method `reload' for nil:NilClass"
+      )
+    end
+  end
```

Comment:
Looks like we lost some other tests that were still meaningful for testing the encode/init changes.